pynubank/nubank.py

```diff
@@ -126,8 +111,9 @@ def get_bills(self):
         request = self.client.get(self.bills_url)
         return request['bills']
 
-    def get_bill_details(self, bill):
-        return self.client.get(bill['_links']['self']['href'])
+    def get_bill_details(self, bill: dict):
+        if '_links' in bill:
```

Comment:
Se não tiver `_links` não retorna nada ? Que tal uma exceção ?

Reply:
Pensando aqui, mas se levantar uma exceção vai dar ruim pros testes né ?

Reply:
Sim, neste caso vai dar ruim nos testes, ao menos nos automatizados, que não esperam excessão. Eu fiz desta forma, pois aqueles testes continuam "genericos". 

Caso a gente insira alguma coisa especificamente para o get_bills vamos ter que mudar nos testes para tratar especificamente este caso naquele teste. 

Na minha visão, o get_bills deve tratar para validar se os dados de entrada são validos e, se não for, passar direto retornando None como é feito atualmente.

Reply:
Entendi, eu acho que seria um comportamento um pouco estranho não retornar nada se a ideia for validar os parametros de entrada.

E no final das contas esse método acaba não sendo testado pelo novo teste né ? (Chama mas ele não vai chamar o `self.client.get`.

E se fizermos uma lista de métodos para ignorarmos (que seria o caso desse `get_bill_details) e criarmos um teste só pra ele ?

Dessa forma testamos que o `MockHttpClient` funciona com todos os métodos e não precisamos alterar o código da aplicação para o teste passar